Formula/mercurial.rb

```diff
@@ -6,6 +6,7 @@ class Mercurial < Formula
   url "https://www.mercurial-scm.org/release/mercurial-6.2.3.tar.gz"
   sha256 "98d1ae002f68adf53d65c5947fe8b7a379f98cf05d9b8ea1f4077d2ca5dce9db"
   license "GPL-2.0-or-later"
+  revision 1
```

Comment:
Revision bumped in case someone already installed `mercurial` and then installs a dependent after `brew update`, which will result in broken formula until `brew reinstall mercurial`.